<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegmentWindow</class>
 <widget class="QMainWindow" name="SegmentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>982</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="viewFrom">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="viewSegmented">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>361</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="viewSelected1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>361</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="viewMerged">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>361</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>301</width>
      <height>951</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>299</width>
       <height>949</height>
      </rect>
     </property>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>281</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Select 1 /2</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>261</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="selectLayout1"/>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>281</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Pre processing</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>29</y>
         <width>261</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="preprocessingLayout"/>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>281</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Segment</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>261</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="splitLayout"/>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>281</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Merge</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>261</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="mergeLayout"/>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>550</y>
        <width>281</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Select 2 /2</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>261</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="selectLayout2"/>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="viewSelected2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>640</y>
      <width>361</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="viewRectangles">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>640</y>
      <width>361</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
